crates/ruff_server/src/session/settings.rs

```diff
@@ -38,6 +38,22 @@ pub(crate) struct ResolvedEditorSettings {
     pub(super) ignore: Option<Vec<RuleSelector>>,
     pub(super) exclude: Option<Vec<String>>,
     pub(super) line_length: Option<LineLength>,
+    pub(super) configuration_resolution_strategy: ConfigResolutionStrategy,
```

Comment:
I'd like to find a way to make the name both shorter and more obvious to users. "Configuration preference"?